app/javascript/legacy_react/app/loading_indicator.ts

```diff
@@ -32,7 +32,7 @@ class PageProgressBar {
   }
 
   private getElem(): Element {
-    return document.getElementById("pageProgressBar")
+    return document.getElementById("pageProgressBar")!
```

Comment:
Are you sure this will never be null? I don't think that's a reasonable conclusion.